packages/vc-export/src/exportToVerifiableCredential.spec.ts

```diff
@@ -1,447 +0,0 @@
-/**
```

Comment:
I removed it because with the changed DID structure it was producing different values and I forgot to generate the new ones. But removing the version number from DID v1 makes them work again.

Reply:
Fixed in https://github.com/KILTprotocol/sdk-js/pull/408/commits/18dfe0b57dd7f164ded8675d24f1da5eb94c23ac